packages/common/src/Constants.js

```diff
@@ -6,11 +6,13 @@ const interfaceName = {
   Store: "Store",
   IdentifierWhitelist: "IdentifierWhitelist",
   CollateralWhitelist: "CollateralWhitelist",
+  AddressWhitelist: "CollateralWhitelist",
   FundingRateStore: "FundingRateStore",
   OptimisticOracle: "OptimisticOracle",
   Bridge: "Bridge",
   GenericHandler: "GenericHandler",
-  MockOracleAncillary: "Oracle"
+  MockOracleAncillary: "Oracle",
```

Comment:
These are added to make the `hardhat` task "setup-finder` (implemented by `common/src/hardhat/tasks/finder.js`) easier to write, doesn't pose any risk to add to this dict IIUC.